lib/verbalize/action.rb

```diff
@@ -45,9 +53,17 @@ def call!
       def input(*required_keywords, optional: [])
         @required_inputs = required_keywords
         optional = Array(optional)
-        @optional_inputs = optional
+        @optional_inputs = optional.reject { |kw| kw.is_a?(Hash) }
```

Comment:
we pass in `optional_inputs` and `default_inputs` separately into `Build.call` because we flatten `optional_inputs` to a list of keys, but default values have stored default lambdas. Trying to conflate them might simplify some interface, but it will greatly complicate all the underlying behavior into having to re-segregate them at multiple other levels. I see why you didn't want this, but honestly I think the interface I built is the appropriate one.